Doc/library/operator.rst

```diff
@@ -17,9 +17,13 @@
 
 The :mod:`operator` module exports a set of efficient functions corresponding to
 the intrinsic operators of Python.  For example, ``operator.add(x, y)`` is
-equivalent to the expression ``x+y``.  The function names are those used for
+equivalent to the expression ``x+y``.  Some function names are those used for
 special class methods; variants without leading and trailing ``__`` are also
-provided for convenience.
+provided for backward compatibility.
```

Comment:
Yes, I have fixed that and also removed the word `also` since it doesn't stand in this context.

Thanks!